### ISSUE TITLE
set up audi booth in artscilab

### ISSUE THREAD

**Issue by rmalina** [artscilab/equipment#9]
ashton

work with brian merlo to set this equipment up
and write the user guide and do a training video on echo 360

roger

Brian

I am approving these purchases
For the
a)	Audio station
b)	B) so we can record skypes with proper mikes and loud speakers

I have checked with frank dufour and he thinks this
Equipment is all compatible with the equipment in the sound
Design studios- 
And I would like Ashton Brillante to work with you
On the installation and at the same time write the user
Guides and training videos

roger

From: Brian Merlo (ATEC at UT Dallas) [mailto:support@atec-utdallas.zendesk.com] 
Sent: Tuesday, September 22, 2015 5:10 PM
To: Malina, Roger
Subject: [ATEC at UT Dallas] Re: RE: ArtSciLab Microphone

##- Please type your reply above this line -##
Your request (153) has been updated. To add additional comments, reply to this email.
 
Brian Merlo (ATEC at UT Dallas) 
Sep 22, 17:10 
Roger, here is the equipment list for the Audio workstation and other miscellaneous items for the lab...
I'm not sure how well zendesk handles web links. I'll send through regular email this is unclear.
Only the top item needs to be purchased with a credit card. The rest can be ordered through the B&H punchout.
-Brian
Articulating monitor arms. -2 
http://www.cotytech.com/content-product_info/product_id-2328/expandable_apple_desk_mount_spring_arm_silver.html
Table stands for studio monitors (speaker stands) -1pair
http://www.bhphotovideo.com/c/product/889973-REG/IsoAcoustics_ISO_L8R200_Studio_Monitor_Stand.html
RCA to to 1/4" cable so that the HDMI and the Jabber camera can be connected to to the Audio system
http://www.bhphotovideo.com/c/search?Ntt=RCA+to+1%2F4&N=0&InitialSearch=yes&sts=ma&Top+Nav-Search=
Microphone for Jabber/Skype meetings
http://www.bhphotovideo.com/c/product/574505-REG/Audio_Technica_AT2050_AT2050_Multi_Pattern_Condenser_Side_Address.html
Webcam for Skype/Hangout meetings 
http://www.bhphotovideo.com/c/product/977620-REG/logitech_960_000971_c930e_webcam_usb.html
Thunderbolt PCIe card
http://www.bhphotovideo.com/c/product/1039491-REG/hp_f3f43at_hp_thunderbolt_2_pcie_1_port_i_o.html
Rechargeable Batteries - 3 sets total, one for each of the podcast kits and one to remain in the lab
3 - Chargers http://www.bhphotovideo.com/c/product/1095851-REG/energizer_chprowb4_recharge_pro_charger_for.html
6 - AABatteries http://www.bhphotovideo.com/c/product/851035-REG/Energizer_NH15BP_4_AA_NiMH_Rechargeable_Batteries.html
1 - AAABateries http://www.bhphotovideo.com/c/product/851032-REG/Energizer_NH12BP_4_AAA_NiMH_Rechargeable_Batteries.html

 
Brian Merlo (ATEC at UT Dallas) 
Sep 16, 16:15 
Request #154 "ArtSciLab Batteries" was closed and merged into this request. Last comment in request #154:
Hey there again,
I know we got a package of AA batteries, but those are already nearly gone. I was wondering if rechargeable batteries had been ordered yet for the ArtSciLab?
Thanks,
Ashton Brillante

 
Malina, Roger 
Sep 16, 10:49 

Amy- you can ignore this
Am resubmitting this to atec tech as it’s a tech support item\
Not a atec support item
Ashton I know its confusing but tech support goes to atec tech
And admin support requests to atec support
roger
From: Brillante, Ashton

Sent: Wednesday, September 16, 2015 10:42 AM

To: ATEC Support

Cc: Malina, Roger

Subject: ArtSciLab Microphone
Hey,
So there microphone set up for our Jabber system is placed on the back wall and during jabber sessions we have sometimes had difficulty hearing people not in close proximity to it. I know someone had mentioned there should be a fix for this. We're just looking to get this issue resolved so we can use Jabber more effectively in the future.
Also, we're looking to record each of our Jabber sessions to be able to review later and possibly use for content. How can we do this?
Thanks,
Ashton Brillante
@cassininazir

**Comment by rmalina** [comment]
ashton\
please contact brian
roger
Brian
Great
I will set up a github item for ashton to work with you
To set up this equipment, get it running and write a small
User guide
roger

From: Brian Merlo (ATEC at UT Dallas) [mailto:support@atec-utdallas.zendesk.com] 
Sent: Friday, October 16, 2015 7:51 PM
To: Malina, Roger
Subject: [ATEC at UT Dallas] Re: RE: ArtSciLab Microphone

##- Please type your reply above this line -##
Your request (153) has been updated. To add additional comments, reply to this email.
 
Brian Merlo (ATEC at UT Dallas) 
Oct 16, 19:51 
Roger, The equipment has been ordered!
Thanks, Brian
P.S. Charlotte, here is the credit card item from below...
Articulating monitor arms. -2 
http://www.cotytech.com/content-product_info/product_id-2328/expandable_apple_desk_mount_spring_arm_silver.html



 
Malina, Roger 
Oct 14, 13:23 
Brian
I am approving these purchases
For the
a) Audio station
b) B) so we can record skypes with proper mikes and loud speakers
I have checked with frank dufour and he thinks this
Equipment is all compatible with the equipment in the sound
Design studios-
And I would like Ashton Brillante to work with you
On the installation and at the same time write the user
Guides and training videos
roger


 
Malina, Roger 
Sep 23, 15:19 
Brian
I think the rechargeable batteries are a no brainer ?
Ashton what do you think
Whilst we review the rest why don’t you go ahead
And order the rechargeable batteries on
My account
roger


 
Brian Merlo (ATEC at UT Dallas) 
Sep 22, 17:10 
Roger, here is the equipment list for the Audio workstation and other miscellaneous items for the lab...
I'm not sure how well zendesk handles web links. I'll send through regular email this is unclear.
Only the top item needs to be purchased with a credit card. The rest can be ordered through the B&H punchout.
-Brian
Articulating monitor arms. -2 
http://www.cotytech.com/content-product_info/product_id-2328/expandable_apple_desk_mount_spring_arm_silver.html
Table stands for studio monitors (speaker stands) -1pair
http://www.bhphotovideo.com/c/product/889973-REG/IsoAcoustics_ISO_L8R200_Studio_Monitor_Stand.html
RCA to to 1/4" cable so that the HDMI and the Jabber camera can be connected to to the Audio system
http://www.bhphotovideo.com/c/search?Ntt=RCA+to+1%2F4&N=0&InitialSearch=yes&sts=ma&Top+Nav-Search=
Microphone for Jabber/Skype meetings
http://www.bhphotovideo.com/c/product/574505-REG/Audio_Technica_AT2050_AT2050_Multi_Pattern_Condenser_Side_Address.html
Webcam for Skype/Hangout meetings 
http://www.bhphotovideo.com/c/product/977620-REG/logitech_960_000971_c930e_webcam_usb.html
Thunderbolt PCIe card
http://www.bhphotovideo.com/c/product/1039491-REG/hp_f3f43at_hp_thunderbolt_2_pcie_1_port_i_o.html
Rechargeable Batteries - 3 sets total, one for each of the podcast kits and one to remain in the lab
3 - Chargers http://www.bhphotovideo.com/c/product/1095851-REG/energizer_chprowb4_recharge_pro_charger_for.html
6 - AABatteries http://www.bhphotovideo.com/c/product/851035-REG/Energizer_NH15BP_4_AA_NiMH_Rechargeable_Batteries.html
1 - AAABateries http://www.bhphotovideo.com/c/product/851032-REG/Energizer_NH12BP_4_AAA_NiMH_Rechargeable_Batteries.html



 
Brian Merlo (ATEC at UT Dallas) 
Sep 16, 16:15 
Request #154 "ArtSciLab Batteries" was closed and merged into this request. Last comment in request #154:
Hey there again,
I know we got a package of AA batteries, but those are already nearly gone. I was wondering if rechargeable batteries had been ordered yet for the ArtSciLab?
Thanks,
Ashton Brillante


 
Malina, Roger 
Sep 16, 10:49 
Amy- you can ignore this
Am resubmitting this to atec tech as it’s a tech support item\
Not a atec support item
Ashton I know its confusing but tech support goes to atec tech
And admin support requests to atec support
roger
From: Brillante, Ashton
Sent: Wednesday, September 16, 2015 10:42 AM
To: ATEC Support
Cc: Malina, Roger
Subject: ArtSciLab Microphone
Hey,
So there microphone set up for our Jabber system is placed on the back wall and during jabber sessions we have sometimes had difficulty hearing people not in close proximity to it. I know someone had mentioned there should be a fix for this. We're just looking to get this issue resolved so we can use Jabber more effectively in the future.
Also, we're looking to record each of our Jabber sessions to be able to review later and possibly use for content. How can we do this?
Thanks,
Ashton Brillante

This email is a service from ATEC at UT Dallas. Delivered by Zendesk 
Message-Id:NY3Z2XKPVJ_56219b8070984_b1e73fbfafecd32c1441d_sprut

**Comment by amb141330** [comment]
I've contacted Brian. Waiting on response.

**Comment by amb141330** [comment]
Will start setting this up soon. Waiting on new bookshelves to make space for the setup.

**Comment by rmalina** [comment]
ah
i see this is still open
ignore my email on why you closed it
roger

On Mon, Nov 2, 2015 at 10:15 AM, amb141330 <notifications@github.com> wrote:

> Will start setting this up soon. Waiting on new bookshelves to make space
> for the setup.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/artscilab/equipment/issues/9#issuecomment-153070165>.
>



-- 
Roger F Malina
is in Dallas

**Comment by amb141330** [comment]
Still waiting on furniture to be supplied before we can start setting this up.

**Comment by amb141330** [comment]
As far as I know furniture is not ready yet.

**Comment by amb141330** [comment]
The audio booth is complete from my end. Brian is adding a second monitor, but other than that it is complete. There is a document on the desktop and in the documents folder giving instructions on how to record a podcast. I know at one of the watering holes they were not able to do so due to an error, but I've been told by Brian this should not happen again.

**Comment by rmalina** [comment]
Can you tell Chaz Lilly where the documents folder with instructions on how
to use avrami Cisco etc
